weave/trace_server/trace_server_interface.py

```diff
@@ -850,6 +897,12 @@ def feedback_query(self, req: FeedbackQueryReq) -> FeedbackQueryRes: ...
     def feedback_purge(self, req: FeedbackPurgeReq) -> FeedbackPurgeRes: ...
 
     # Action API
+    def actions_execute_batch(
+        self, req: ActionsExecuteBatchReq
+    ) -> ActionsExecuteBatchRes: ...
+    def actions_ack_batch(self, req: ActionsAckBatchReq) -> ActionsAckBatchRes: ...
```

Comment:
`actions_ack_batch` probably belongs on the `ClickhouseTraceServer`

Reply:
done